lib/ansible/modules/cloud/vmware/vmware_host_config_info.py

```diff
@@ -78,20 +79,20 @@
 from ansible.module_utils.vmware import vmware_argument_spec, PyVmomi
 
 
-class VmwareConfigFactsManager(PyVmomi):
+class VmwareConfigInfoManager(PyVmomi):
     def __init__(self, module):
-        super(VmwareConfigFactsManager, self).__init__(module)
+        super(VmwareConfigInfoManager, self).__init__(module)
         cluster_name = self.params.get('cluster_name', None)
         esxi_host_name = self.params.get('esxi_hostname', None)
         self.hosts = self.get_all_host_objs(cluster_name=cluster_name, esxi_host_name=esxi_host_name)
 
-    def gather_host_facts(self):
+    def gather_host_info(self):
         hosts_facts = {}
         for host in self.hosts:
-            host_facts = {}
+            host_fact = {}
```

Comment:
```suggestion
            host_info = {}
```
And the 2 usages below.